src/main/java/cart/application/product/ProductService.java

```diff
@@ -0,0 +1,48 @@
+package cart.application.product;
+
+import cart.domain.product.Product;
+import cart.dto.product.ProductRequest;
+import cart.dto.product.ProductResponse;
+import cart.repository.product.ProductRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public List<ProductResponse> getAllProducts() {
+        List<Product> products = productRepository.findAll();
+        return products.stream().map(ProductResponse::of).collect(Collectors.toList());
```

Comment:
```suggestion
        return products.stream()
                .map(ProductResponse::of)
                .collect(Collectors.toList());
```

가독성 및 디버깅을 위해 `stream` 문은 `.`단위로 개행하면 어떨까요?

Reply:
수정된 방식이 가독성이 더 높은 것 같습니다 수정하겠습니다!